commands/credentials/repair_default.js

```diff
@@ -8,12 +8,17 @@ function * run (context, heroku) {
   const host = require('../../lib/host')
   const util = require('../../lib/util')
 
-  const {app, args} = context
+  const {app, args, flags} = context
 
   let db = yield fetcher.addon(app, args.database)
   if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
 
-  yield cli.action(`Resetting permissions for default role to factory settings`, co(function * () {
+  yield cli.confirmApp(app, flags.confirm, `WARNING: Destructive Action
+This command will give the default credential ownership of all database objects owned by additional credentials. 
```

Comment:
Should we be more explicit that it's removing ownership from the other credentials? I.e., something like "Ownership of all database objects owned by additional credentials will be transferred to the default role"?